examples/benchmark/main.go

```diff
@@ -72,7 +72,7 @@ func run() error {
 	}
 
 	accountKey := keypair.MustRandom()
-	escrowAccountKey := keypair.MustRandom()
+	multiSigAccountKey := keypair.MustRandom()
```

Comment:
We should use `multisig` all lower case throughout, for the same reasons we used lower case in the SEP.

Reply:
https://github.com/stellar/starlight/pull/408/commits/9afa4fc8779a5b22c6d22c2580330366c548857d 👍

Reply:
changed `MultiSig` to `Multisig` here:
https://github.com/stellar/starlight/pull/408/commits/533547ead48c6dd883f31d32ee520b6fd8f2cc8e